src/views/GameListView.jsx

```diff
@@ -90,27 +96,32 @@ const GameListView = () => {
       if (!mounted) return;
 
       setGameList(
-        content.sort((a, b) => {
-          const [nameA, nameB] = [
-            a.board.defaultName || a.board.name,
-            b.board.defaultName || b.board.name,
-          ];
-          if (nameA < nameB) {
-            return -1;
-          }
-          if (nameA > nameB) {
-            return 1;
-          }
-          return 0;
+        content.filter((game) => {
```

Comment:
I think this filter should be applied in a `useMemo` outside of this `useEffect` to avoid  calling the api each time the search terms changes.

Reply:
100%.